src/DoctrineGui/Form/ClientFieldset.php

```diff
@@ -49,6 +49,20 @@ public function init()
         );
 
         $this->add([
+                'name'          => 'user',

```

Comment:
Yes, like:
```
'key' => 'value',
'otherkey => 'othervalue',
```
The way to do this is not defined in PSR1 or PSR2.  It is defined in Symfony that you should align them like
```
'key'     => 'value',
'otherkey' => 'othervalue',
```
so it's your choice of style, I just want it always applied or never applied, not a mix.